plinth/templates/index.html

```diff
@@ -70,6 +70,15 @@ <h3 class="panel-title">{{ details_label }}</h3>
           {{ paragraph|safe }}
         </div>
       {% endfor %}
+
+      {% if user.is_authenticated %}
+        {% for paragraph in configure_url %}
```

Comment:
If we change configure_url to not be a list (as I suggested above), then we won't need the for loop here.